_rules/SC2-4-9-links-with-identical-accessible-names-have-same-purpose.md

```diff
@@ -0,0 +1,233 @@
+---
+name: Links with identical accessible names have same purpose
+
+description: |
+  This rule checks that identical accessible names are only used for links that have the same purpose
+
+success_criterion: 
+- 2.4.9 # Link Purpose (Link Only)
+
+test_aspects:
+- DOM Tree
+- CSS Styling
+
+authors:
+- Anne Thyme Nørregaard
+---
+
+## Test Procedure
+
+### Applicability
+
+The rule applies to any two or more HTML or SVG elements that have the [semantic role](#semantic-role) of link, are [exposed to assistive technologies](#exposed-to-assistive-technologies) and that have the same [accessible name](#accessible-name).
+
+**Note:** Leading and trailing whitespace and difference in case sensitivity should be ignored when deciding whether the [accessible names](#accessible-name) are the same.
```

Comment:
This seems to be implementation detail and may be it is best if it lived within the definition of accessible name?

Reply:
Put into seperate PR since other rules are also relying on the definition: https://github.com/auto-wcag/auto-wcag/pull/254